src/app/core/config-setup/config-import-parser.service.ts

```diff
@@ -0,0 +1,144 @@
+import { Injectable } from "@angular/core";
+import { EntityConfig } from "../entity/entity-config.service";
+import { EntityListConfig } from "../entity-components/entity-list/EntityListConfig";
+import { EntityDetailsConfig } from "../entity-components/entity-details/EntityDetailsConfig";
+import {
+  CONFIGURABLE_ENUM_CONFIG_PREFIX,
+  ConfigurableEnumConfig,
+} from "../configurable-enum/configurable-enum.interface";
+import { EntitySchemaField } from "../entity/schema/entity-schema-field";
+import { ConfigService } from "../config/config.service";
+import { ConfigFieldRaw } from "./config-field.raw";
+
+@Injectable({
+  providedIn: "root",
+})
+export class ConfigImportParserService {
+  enumsAvailable: Map<string, ConfigurableEnumConfig> = new Map();
+  existingEnumHashmap: Map<string, string> = new Map();
+
+  constructor(private configService: ConfigService) {
+    const enums = this.configService.getAllConfigs(
+      CONFIGURABLE_ENUM_CONFIG_PREFIX
+    );
+    // TODO: how to get the id?
+  }
+
+  parseImportDefinition(
+    configRaw: ConfigFieldRaw[],
+    entityName: string
+  ): GeneratedConfig {
+    const entity: EntityConfig = {
+      attributes: configRaw
+        .filter((field) => !!field.dataType)
+        .map((field) => this.parseFieldDefinition(field)),
+    };
+
+    const generatedConfig: GeneratedConfig = {
+      ["entity:" + entityName]: entity,
+    };
+
+    // add enum configs
+    for (const [key, enumConfig] of this.enumsAvailable) {
+      generatedConfig["enum:" + key] = enumConfig;
```

Comment:
```suggestion
      generatedConfig[key] = enumConfig;
```
If the approach from above is used